client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -849,6 +877,17 @@ export default {
       return this.showPageNavigation || this.options.showTotalCount
     },
 
+    rowOptions () {
+      return [
+        { text: this.options.perPage, value: this.options.perPage, disabled: true },
+        { text: '25', value: 25 },
+        { text: '50', value: 50 },
+        { text: '100', value: 100 },
+      ]
+      // remove duplicates
+        .filter((v, i, a) => a.findIndex(t => (t.value === v.value)) === i)
```

Comment:
You can just check if option is the same as the options.perPage, and index is not the first one
it should also be sorted, since you don't know if default option is the lowest value

Reply:
This still needs to be resolved
Also, if the this.options.perPage value is 0, then label should be 'All' and be last in the array.